### ISSUE TITLE
Getting assembley reference exceptions in the WebDriver 4 Alpha2 version in DotNet bindings when using .NetStandard or .NetCore projects

### ISSUE THREAD

**Issue by okyrylenko** [SeleniumHQ/selenium#7436]
## 🐛 Bug Report

Getting reference exception to these 2 packages:
System.Private.CoreLib 
System.Private.Uri

I manually installed these assembleys but still was having this exception
Worked fined when downgraded to the alpha1 version

Even though the screenshot was taken on Mac, it also does happen on Windows platform. 
Having this issue .NetCore and .NetStandard Works fine in .NetFramework
-->

## To Reproduce

1. Create .NetCore or .NetStandard class library project
2. Add Selenium.WebDriver, Selenium.Support alpha2 packages and latest nUnit package
3. Compile

Expected: compiles without the issues
Actual: getting package reference exception even after adding those packages manually

![1](https://user-images.githubusercontent.com/40716729/62001856-bd27e680-b0c7-11e9-9d78-44267c4f8907.png)

**Comment by mcopjan** [comment]
The same, can't install this version of package into my netstandard2.0 project

**Comment by RemiBou** [comment]
Same here, for reproduction  : 
- install sdk 3 last preview
- git clone https://github.com/RemiBou/Toss.Blazor.git
- cd Toss.Blazor/Toss.Tests.E2E
- Edit Toss.Tests.E2E.csproj and upgrade WebDriver from alpha01 to alpha02
- dotnet build

**Comment by jimevans** [comment]
This issue was caused by the move to using Bazel to build the .NET bindings. This will take some work to resolve, I'm afraid. Thank you for the report; your patience is appreciated.

**Comment by jimevans** [comment]
For what it's worth, I've submitted [an issue](https://github.com/bazelbuild/rules_dotnet/issues/142) to the Bazel .NET build rules project concerning this. I don't expect a quick resolution, which means I'll need to reconfigure the build system for the next alpha release.

**Comment by okyrylenko** [comment]
@jimevans  thanks Jim, do you want to close this bug or keep it open until Bazel will fix it on their end?

**Comment by Dreamescaper** [comment]
Seems like issue still present in alpha-4.
Now it builds successfully, but fails in runtime with:
```
System.IO.FileNotFoundException: Could not load file or assembly 'netstandard2.0assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
   at WebDriverRepro.Program.Main(String[] args)
```


[WebDriverRepro.zip](https://github.com/SeleniumHQ/selenium/files/4049575/WebDriverRepro.zip)

**Comment by Chryogenic** [comment]
Any updates on this isssue?

**Comment by Dreamescaper** [comment]
Probably should be closed, as fixed in alpha05.

**Comment by jimevans** [comment]
Closing as fixed by 72f9d42
